canonical/microk8s.io
# Apply redesign to nav and footer

## Done

- Update to latest global nav version
- Apply redesign to nav and footer as per Jira ticket 

## QA

- View the site locally in your web browser at: https://microk8s-io-614.demos.haus/
- Check nav and footer against design
- Make sure all links work as expected **(/features will 404, this will be fixed once https://github.com/canonical/microk8s.io/pull/613 is merged)**


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5794
